plugins/de.fraunhofer.ipa.ros.xtext/src/de/fraunhofer/ipa/ros/Ros.xtext

```diff
@@ -72,44 +73,43 @@ Artifact returns Artifact:
 
 Node returns Node:
     'node:' name=RosNames
-     BEGIN
```

Comment:
was this "BEGIN" removed intentionally? why?
The interfaces should be defined under the node.